fe2-android/app/src/main/java/com/github/dedis/student20_pop/model/data/LAORepository.java

```diff
@@ -514,6 +517,14 @@ public static void destroyInstance() {
       CreateLao data = (CreateLao) message.getData();
       createLaoRequests.put(id, "/root/" + data.getId());
     }
+    // Uncomment to test display without message from Backend
+    else {
+      if(message.getData() instanceof ElectionSetup) {
+        handleElectionSetup(channel,(ElectionSetup) message.getData());
+      }
+    }
```

Comment:
This "testing hack" for the display should be deleted, this way the election will be handled twice when we receive the broadcast from the backend